aioesphomeapi/connection.pxd

```diff
@@ -0,0 +1,58 @@
+import cython
+
+cdef dict MESSAGE_TYPE_TO_PROTO
+cdef dict PROTO_TO_MESSAGE_TYPE
+
+cdef set OPEN_STATES
+
+cdef float KEEP_ALIVE_TIMEOUT_RATIO
+
+cdef bint TYPE_CHECKING
+
+cdef object DISCONNECT_REQUEST_MESSAGE
+cdef object PING_REQUEST_MESSAGE
+cdef object PING_RESPONSE_MESSAGE
+
+cdef object DisconnectRequest
+cdef object PingRequest
+cdef object GetTimeRequest
+
```

Comment:
I kept this pxd simple and only added types as I didn't want to make it difficult for anyone else to maintain.